pytrip/vdx.py

```diff
@@ -933,7 +932,7 @@ def create_dicom_contour_data(self, i):
             contours.extend(_slice.create_dicom_contours(dcmcube))
 
         roi_contours.ContourSequence = Sequence(contours)
-        roi_contours.ROIDisplayColor = self.get_color(i)
+        roi_contours.ROIDisplayColor = self.get_color()
```

Comment:
It seems that `create_dicom_contour_data` (see few lines above) has now obsolete argument `i`

Reply:
Removed